ksql-engine/src/test/java/io/confluent/ksql/function/udf/datetime/TimestampToStringTest.java

```diff
@@ -46,6 +46,50 @@ public void shouldCovertTimestampToString() {
     assertThat(result, is(expectedResult));
   }
 
+  @Test
+  public void testUTCTimeZone() {
+    // When:
+    final Object result = udf.evaluate(1534353043000L, "yyyy-MM-dd HH:mm:ss", "UTC");
+
+    // Then:
+    assertThat(result, is("2018-08-15 17:10:43"));
+  }
+
+  @Test
+  public void testPSTTimeZone() {
+    // When:
+    final Object result = udf.evaluate(1534353043000L,
+        "yyyy-MM-dd HH:mm:ss", "America/Los_Angeles");
+
+    // Then:
+    assertThat(result, is("2018-08-15 10:10:43"));
+  }
+
+  @Test
+  public void testTimeZoneInFormat() {
+    // When:
+    final long timestamp = 1534353043000L;
+    final Object localTime = udf.evaluate(timestamp,
+        "yyyy-MM-dd HH:mm:ss zz");
+    final Object pacificTime = udf.evaluate(timestamp,
+        "yyyy-MM-dd HH:mm:ss zz", "America/Los_Angeles");
+    final Object universalTime = udf.evaluate(timestamp,
+        "yyyy-MM-dd HH:mm:ss zz", "UTC");
+
+    // Then:
+    final String expected = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss zz")
+        .format(new Date(timestamp));
+
+    assertThat(localTime, is(expected));
+    assertThat(pacificTime, is("2018-08-15 10:10:43 PDT"));
+    assertThat(universalTime, is("2018-08-15 17:10:43 UTC"));
+  }
+
+  @Test(expected = KsqlFunctionException.class)
+  public void shouldThrowIfInvalidTimeZone() {
+    udf.evaluate(1638360611123L, "yyyy-MM-dd HH:mm:ss.SSS", "PST");
```

Comment:
Can you use Junit's `ExpectedException` to check the message includes all params. Also test with only two params